lib/Ohmbrewer_Temperature_Sensor.cpp

```diff
@@ -212,3 +311,11 @@ void Ohmbrewer::TemperatureSensor::whichPins(std::list<int>* pins) {
 
 }
 
+/*
+ * publish to the log
+ */
+void log(char* msg)
```

Comment:
somehow this function magically disappeared from my code , my file ends at line 288.....weird